adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -128,10 +145,19 @@ public void transform(StructuredRecord input, Emitter<KeyValue<AvroKey<GenericRe
     @Macro
     public String fileSystemProperties;
 
-    @Description("Output schema for the output files.")
+    @Nullable
+    @Description("Output schema for avro format output files.")
```

Comment:
it's not just for avro. Should add information about what it means for text and what are invalid types when using text.